content/_code-samples/checks/js/cancel_check.js

```diff
@@ -0,0 +1,36 @@
+const xrpl = require('xrpl');
+
+const secret = "sEdTPPEeMH6SAgpo6rSj8YW7a9vFfUj";
+const checkId = "505E0A4CCBCC971EE07DCD25611A612830CFFA4D50DBC04947EF952D898C94F3"; // Replace with your check ID
```

Comment:
nit: Having an example checkId prefilled does not make sense to me because the likelihood of it existing is almost 0 because it gets destroyed after one run.  Warn the user if its empty to be more user friendly. 

I understand the original version had it like did it.

Reply:
Updated the code as per suggestion